### ISSUE TITLE
High CLS on page resize

### ISSUE THREAD

**Issue by ramironieto** [GoogleChrome/lighthouse#14396]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

https://web.dev/

### What happened?

When running lighthouse on timespan mode and resizing a page it shows high CLS.

### What did you expect?

it should be low as it is within the time window of a user-initiated layout shift, and the resizing is done by the user.

### How were you running Lighthouse?

Chrome DevTools

### Lighthouse Version

9.6.2

### Chrome Version

105.0.0.0

### OS

Mac

**Comment by adamraine** [comment]
Never-mind, I needed to resize the window not just the inspector pane. I can repro this.

**Comment by adamraine** [comment]
We ignore the `had_recent_input` flag for the first set of layout shifts
https://github.com/GoogleChrome/lighthouse/blob/0ce84c0feb444964dd8fe1555230e1efd785993d/core/computed/metrics/cumulative-layout-shift.js#L24-L29

Instead of the current implementation, maybe we should do this for any layout shifts that occur within the first X ms of the trace starting.

**Comment by connorjclark** [comment]
isn't that when shifting is most likely to happen?

**Comment by adamraine** [comment]
> isn't that when (real, bad) shifting is most likely to happen?

Yeah, that's why we want to ignore `had_recent_input` for those events. I'm proposing this logic will stay the same for LS events within the first 500ms of tracing.

If every LS event is triggered by some user input then the current logic will count them all.

**Comment by connorjclark** [comment]
Ah, right. The negation here tripped me up. But why would a manual resize be limited to just the first X ms? You can resize a the windows during devtools anytime.

**Comment by adamraine** [comment]
>  But why would a manual resize be limited to just the first X ms? 

We are concerned with the Lighthouse emulation resize at the start of the run being treated as a user input by LS events. Any LS event with `had_recent_input = true` within the first 500ms of tracing could have been set with `had_recent_input = true` because of the Lighthouse emulation resize.

[LS events only look for user input in a 500ms window before the LS](https://github.com/WICG/layout-instability#recent-input-exclusion:~:text=The%20hadRecentInput%20attribute%20is%20true%20when%20the%20last%20input%20occurred%20within%20the%20past%20500%20ms). This is where the 500ms figure comes from.

**Comment by adamraine** [comment]
https://bugs.chromium.org/p/chromium/issues/detail?id=1302667
